mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Argh. Where does it say that the 2000 has a normal vector in the Z direction? That is contrary to what MaxFilter / `coil_def` is saying, right?